_data/toc/php-developer-guide.yml

```diff
@@ -214,6 +214,9 @@ pages:
               include_versions: ["2.1", "2.2", "2.3"]
               url: /extension-dev-guide/message-queues/queue-migration.html
 
+            - label: Handling outdated in-memory object states
```

Comment:
Move the new topic so that it's before the migration topic. The migration topic is semi-obsolete for 2.3